src/main/kotlin/ai/digital/integration/server/common/tasks/cluster/operator/OperatorBasedUpgradeClusterTask.kt

```diff
@@ -234,8 +234,6 @@ abstract class OperatorBasedUpgradeClusterTask(@Input val productName: ProductNa
     private fun launchCluster(operatorHelper: OperatorHelper) {
         if (operatorHelper is AwsOpenshiftOperatorHelper) {
             operatorHelper.launchCluster()
-        } else if (operatorHelper is AwsOpenshiftHelmHelper) {
-            operatorHelper.launchCluster()
```

Comment:
why aws openshifthelmhelper is removed

Reply:
Reverting this one change - Removed by AI